roadmap.md

```diff
@@ -12,6 +12,23 @@ This includes the creation of end user apps, new platform functions or target im
 
 ## Milestones
 
-The PELUX project targets milestone releases twice per year. Our first target is October 2017.
+The PELUX project targets to provide biannual releases. The first target being September 2017.
 
-The exact contents of the October 2017 release is still TBD.
+###PELUX 1.0 [September 2017]
```

Comment:
Insert a space here between `###` and `PELUX`, otherwise it doesn't render properly